front/src/applications/osrd/components/Helpers/ChartHelpers.ts

```diff
@@ -268,12 +305,12 @@ export const interpolateOnPosition = (dataSimulation, keyValues, positionLocal)
 
 // Interpolation of cursor based on time position
 export const interpolateOnTime = (
-  dataSimulation: Regime,
+  dataSimulation: { [key: string]: any } | undefined,
```

Comment:
Regime is correct here.
listValues is an `Array<keyof Regime>`

for example a call  in `src/applications/osrd/views/OSRDSimulation/Map.tsx`
![Screenshot_20221108_130854](https://user-images.githubusercontent.com/24652545/200560131-87481107-a3d6-4d4e-877a-ab3a0f80a121.png)

Reply:
`interpolateOnTime` is also used here : https://github.com/DGEXSolutions/osrd/blob/dev/front/src/reducers/osrdsimulation/index.ts#L323 which is a `SimulationTrain` not a Regime.